### PR TITLE
fix(core): fix split-button accessibility

### PR DESCRIPTION
**Pull request by dimamarksman**
## Related Issue(s)

closes https://github.com/SAP/fundamental-ngx/issues/6185

## Description

This PR fixes core split-button accessibility

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [x] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

File: libs/core/src/lib/split-button/split-button.component.html [hunk 1]
```diff
@@ -1,4 +1,11 @@
-<div class="fd-button-split fd-has-margin-right-small" role="group" aria-label="button-split" [fdMenuTrigger]="menu">
+<div
+    class="fd-button-split fd-has-margin-right-small"
+    role="group"
+    aria-label="Split button"
```
Comment (Betrozov) [inline libs/core/src/lib/split-button/split-button.component.html:4]: i18n?

**Review by Betrozov — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/button/button.component.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ import { applyCssClass } from '@fundamental-ngx/core/utils';
         '[attr.type]': 'type',
         '[attr.disabled]': '_disabled || null',
         '[attr.aria-label]':
-            'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label != null ?  label +" ,"+ fdType : glyph != undefined || glyph != null ? fdType +","+ glyph.split("-").join(" ")  : title : title'
+            'ariaLabel ? ariaLabel : specialButtonType.includes(fdType) ? label != null ? label + ", " + fdType : glyph != undefined || glyph != null ? fdType + ", " + glyph.split("-").join(" ") : title : title'
```
Comment (platon-rov) [inline libs/core/src/lib/button/button.component.ts:42]: Could you please make it more readable?

Comment (dimamarksman) [reply libs/core/src/lib/button/button.component.ts]: moved it to a getter

**Review by platon-rov — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

Issue 1: Fixed.

Issue 2: Fixed.

Issue 3: Fixed.

Retest Details:
https://deploy-preview-7239--fundamental-ngx.netlify.app/#/core/split-button
JAWS 2021

**Review by puru-hk — APPROVED** [review]

**Pull request merged**